client/ayon_core/tools/attribute_defs/files_widget.py

```diff
@@ -892,6 +892,29 @@ def _on_split_request(self):
             self._add_filepaths(new_items)
         self._remove_item_by_ids(item_ids)
 
+    def _on_merge_request(self):
```

Comment:
BTW This should be ignored if the definition does not allow sequences.

And the action should also not be shown.